src/arch/xtensa/xtos/CMakeLists.txt

```diff
@@ -11,10 +11,7 @@ set(VECTOR_DEFS
 	-D__SPLIT__level5
 )
 
-if(CONFIG_APOLLOLAKE
-   OR CONFIG_CANNONLAKE
-   OR CONFIG_SUECREEK
-   OR CONFIG_ICELAKE)
+if(CONFIG_LEVEL6_IRQ)
```

Comment:
I think that ~20lines later you may also need to append 6 to `set(levels 2 3 4 5)`

Reply:
I assumed it is used, but now I see we don't have level 6. I will double check and possibly remove it at all.

Reply:
Yea, these defines for lvl6 were there from autoconf times, but it looked that no1 was using it anyway. APL+ / CNL+ had defines but only Suecreek built lvl6 lib actually, but never included it into final binary tho.